2018/ffs-wp.md

```diff
@@ -0,0 +1,277 @@
+- K4: Materialkosten (für T-Shirts, etc) wird zusätzlich berechnet
+
+
+Die Zuteilung der Gelder wird "crypto-transparent" veröffentlicht:
+
+- Beim Bezahlen bekommt man einen Token.
+- Alle verkauften Support-Zertifikate werden veröffentlicht. Enthaltene Daten:
+    - Token
+    - Gesamtsumme
+    - K1-Zuteilungsschema
+- -> Jede\*r Käufer\*in kann sich überzeugen, ob das eigene Zertifikat korrekt in der Gesamtbilanz enthalten ist (partielle Geldflüsse).
+- Alle können sich überzeugen, dass die Gesamtbilanz aufgeht.
+- Da die verkauften Zertifikate signiert sind, kann die FFS sich vor Anschuldigung der Unterschlagung schützen.
+
+
+**Was passiert mit dem Geld bzw. durch das Geld?**
+
+- Die registrierten Projekte können
+    - sich das Geld auszahlen lassen.
+    - das Geld transparent an andere Projekte weiterleiten (z.B. zu libs von denen sie abhängen)
+- Formal gesehen sind es Spendengelder.  Die Verwendung liegt im Ermessen der Empfänger\*innen.
+Die Projekte verpflichten sich bei der Registrierung in jedem Intervall einen Bericht zur Verwendung der Gelder zu veröffentlichen. Beispiele:
+    - "Konnte es mir leisten, meine Wochenarbeitszeit zu reduzieren und mehr Zeit in Projekt XYZ zu investieren."
+    - "Von dem Geld wurden eine Anzeigenkampagne (Print- und Online) bezahlt. Die Downloadzahlen haben sich in dem Zeitraum verdreifracht. Wir wollen das wieder machen."
+    - "Durch das Geld konnten wir zwei Mitarbeiter\*innen aus dem Usability Team zu je 20% Vollzeit auf das Projekt ansetzen. Und eine studentische Hilfskraft hat die Doku aktualisiert."
+    - "Wir haben Firma XYZ Beauftragt Issue 123 zu schließen. Für die Zukunft planen wir das gleiche mit Issue 456."
+    - "Ich sehe das Geld als Anerkennung für die im letzten Jahr geleistete Arbeit."
+        - Ähnelt dem Bounty-Modell (siehe oben).
+    - "Im letzten Quartal wurden hauptsächlich Issues bearbeitet, die von Spender\*innen hoch bewertet wurden."
+        - Das entspricht dem Reverse-bounty-Modell.
+
+**Wird dafür eine eigene Organisation gebraucht?**
+
+- Für einzelne Projekte ist der Aufwand zu groß.
+- Ggf. könnte eine bestehende Organisation (naheliegend: Free Software Foundation oder FSFE) die Idee umsetzen. Das wäre aber optional.
+- Durch den einkalkulierten Betriebskosten-Anteil könnte sich das FFS-Projekt selbst tragen, ggf. sogar als klassisches StartUp mit Risiko-Kapital.
+- Vorteil einer eigenen Organisation: Agilität
+
+
+
+**Potentielle Probleme des Konzeptes (und Gegenstrategien)**
+
+- Es gibt schon ähnliche Versuche (siehe Links unten) ohne durchschlagenden Erfolg.
+    - Die zweite Maus bekommt den Käse...
+- Kritische Masse wird benötigt (=Anschubfinanzierung für Marketing)
+    - Ggf. durch Crowdfunding oder öffentliche Förderung.
+- Gefahren durch Kommerzialisierung von FLOSS-Entwicklung
+    - Auch jetzt gibt es schon Geld-Ströme (z.B. Google-Summer-of-Code, Bounties, ...)
+- Registrierung und Bezahlprozess zu aufwendig?
```

Comment:
führt zu Zentralisierung (paypal, Patreon, flattr etc.)

Reply:
Zentralisierung ist nicht immer was schlechtes. Von den genannten Services finde ich drei "Ok".

Reply:
Hmm. Ist ein Punkt. Versuch das zu entkräften
- Transparenz und Datensparsamkeit von Anfang zentral.
- Wenn das Projekt groß genug würde, um relevant Macht auszuüben ergibt sich mindestens ein Fork.

Reply:
Wie im vorherigen Kommentar angesprochen: die Alternativen parallel anbieten => nicht mehr zentral. Aber ihr meint jetzt wahrscheinlich die FFS als „Zentrale“; aye, dazu könnte man ab einer ausreichenden Größe eine Aufspaltung mit Themenschwerpunkten andenken. Es gibt ja bereits diverse Stiftungen, die ähnliches tun (Mozilla, p≡p, FSF, EFF … ihr kennt sicher mehr).

Reply:
Man könnte sogar in der Satzung festlegen, "wenn das Volumen x Euro übersteigt, wird ein Fork-Prozess eingeleitet."

Halte ich aber insgesamt für ein Implementierungsdetail, was *jetzt* noch nicht relevant ist.